net-analyzer/wireshark/wireshark-9999.ebuild

```diff
@@ -68,7 +68,6 @@ RDEPEND="
 	gui? (
 		dev-qt/qtbase:6[concurrent,dbus,gui,widgets]
 		dev-qt/qt5compat:6
-		dev-qt/qtdeclarative:6
```

Comment:
Was a bit worried about this but at a glance I indeed can't see any Qml use, so hopefully is fine.